src/main/java/leo/task/Task.java

```diff
@@ -0,0 +1,201 @@
+package leo.task;
+
+import java.io.IOException;
+import java.io.Serializable;
+import java.time.format.DateTimeFormatter;
+import java.time.LocalDateTime;
+import java.util.Arrays;
+import java.util.List;
+
+import leo.ui.Ui;
+
+public class Task implements Serializable {
+    private static final long serialVersionUID = 1L;
+    
+    enum Type {
+        TODO, DEADLINE, EVENT
+    };
+
+    private String taskDesc;
+    private boolean isDone = false;
+    private Type type;
+    public final static List<String> descCommands = Arrays
```

Comment:
Hi, I think this variable name can be more descriptive, when I first saw it I thought it was short for describeCommands but it seems like it is short for descriptionCommands.

Was thinking you can put your commands as an enum, with a tag to check if it has any descriptions.
Something like:
```java
public enum Command {
    TODO("todo", true),
    LIST("list", false);

    private final String command;
    private final boolean desc;

    Command(String command, boolean desc) {
        this.command = command;
        this.desc = desc;
    }

    public String getCommand() {
        return command;
    }

    public int getDesc() {
        return desc;
```

Reply:
This is fantastic, thank you